models/project/project.go

```diff
@@ -115,6 +199,25 @@ func (u *ProjectStruct) Lists() []*list.ListStruct {
 	return slice
 }
 
+func (u *ProjectStruct) NoneList() *list.ListStruct {
+	table := u.database.Init()
+	defer table.Close()
+
+	var id, projectId, userId int64
```

Comment:
__[golint]__
* [golint] var projectId should be projectID
* [golint] var userId should be userID